internal/scorer/config_test.go

```diff
@@ -0,0 +1,214 @@
+package scorer
```

Comment:
Can you add the copyright/license blob at the top of this file too, please.

Reply:
Done!